app/scripts/controllers/console.js

```diff
@@ -232,5 +281,107 @@ angular.module('console', ['sql'])
       }
     };
   })
+  .directive('lazyLoadScroll', function ($window) {
+    return {
+      restrict: 'A',
+      link: function (scope, element, attrs) {
+        var e = element[0];
+        $($window).bind('scroll', function () {
```

Comment:
this looks a bit weird because you are injecting a $window to use as a jquery selector.. you can use `window` directly as a selector in jquery and you already have access to .scroll() event handler https://api.jquery.com/scroll/ it looks like .bind() is deprecated